### ISSUE TITLE
why is TestMMTest.ts not valid pascal case? (`unicorn/filename-case`)

### ISSUE THREAD

**Issue by salisbury-espinosa** [sindresorhus/eslint-plugin-unicorn#2741]
```bash
TestMMTest.ts
  1:1   error    Filename is not in pascal case. Rename it to `TestMmTest.ts`  unicorn/filename-case
```
why?
`MM` - an abbreviation of two words beginning with `M`,

example of correct validation
https://github.com/sindresorhus/type-fest/blob/main/source/pascal-case.d.ts
```typescript
import type { PascalCase } from 'type-fest';

// TestMMTest - CORRECT!!!!!!
const someVariable: PascalCase<'test-MM-test'> = 'TestMMTest';
```

**Comment by sindresorhus** [comment]
The general convention in JS is to treat acronyms in names as whole words, i.e. use `loadHttpUrl`, not `loadHTTPURL`, unless required by a platform name (e.g.
  `XMLHttpRequest`)
